receiver/vcenterreceiver/metadata.yaml

```diff
@@ -0,0 +1,533 @@
+name: vmwarevcenterreceiver
+
+resource_attributes:
+  vcenter.cluster.name:
+    description: The name of the vCenter Cluster
+    type: string
+  vcenter.host.name:
+    description: The hostname of the vCenter ESXi host
+    type: string
+  vcenter.resource_pool.name:
+    description: The name of the resource pool
+    type: string
+  vcenter.datastore.name:
+    description: The name of the vCenter datastore
+    type: string
+  vcenter.vm.name:
+    description: The name of the virtual machine
+    type: string
+  vcenter.vm.id:
+    description: The instance UUID of the virtual machine
+    type: string
+
+attributes:
+  disk_state:
+    description: The state of storage and whether it is already allocated or free.
+    type: string
+    enum:
+      - available
+      - used
+  host_effective:
+    value: effective
+    description: Whether the host is effective in the vCenter cluster
+    type: bool
+    enum:
+      - "true"
+      - "false"
+  latency_direction:
+    value: direction
+    description: The direction of disk latency.
+    enum:
+      - read
+      - write
+  latency_type:
+    value: type
+    description: The type of disk latency being reported.
+    enum:
+      - kernel
+      - device
+  throughput_direction:
+    value: direction
+    description: The direction of network throughput.
+    type: string
+    enum:
+      - transmitted
+      - received
+  vm_count_power_state:
+    value: power_state
+    description: Whether the virtual machines are powered on or off
+    type: string
+    enum:
+      - "on"
+      - "off"
+  vsan_latency_type:
+    value: type
+    description: The type of vSAN latency.
+    type: string
+    enum:
+      - read
+      - write
+  vsan_operation_type:
+    value: type
+    description: The type of vSAN operation.
+    type: string
+    enum:
+      - read
+      - write
+      - unmap
+  vsan_throughput_direction:
+    value: direction
+    description: The type of vSAN throughput.
+    type: string
+    enum:
+      - read
+      - write
+
+metrics:
+  vcenter.cluster.cpu.available:
+    enabled: true
+    description: The amount of CPU available to the cluster
```

Comment:
Is this a fixed value? If so, I think it is a `limit` metric

Reply:
Yep it does have potential to grow so I think you are right to make it a `.limit` metric 👍